Examples/UIExplorer/NavigationExperimental/NavigationAnimatedExample.js

```diff
@@ -105,7 +110,7 @@ const styles = StyleSheet.create({
     flex: 1,
   },
   scrollView: {
-    marginTop: 64
+    marginTop: Platform.OS === 'ios' ? 64 : 56,
```

Comment:
Could we import `STATUSBAR_HEIGHT` from the header module? (same comment to the other examples)